liquigraph-core/src/test/java/org/liquigraph/core/validation/XmlSchemaValidatorTest.java

```diff
@@ -104,11 +104,30 @@ public void parses_changelog_with_explicit_schema() throws Exception {
         assertThat(validator.validateSchema(asNode("changelog/changelog-with-schema-location.xml"))).isEmpty();
     }
 
+    @Test
+    public void parses_changelog_with_parameters() throws Exception {
+        assertThat(validator.validateSchema(asNode("changelog/parameterized-changelog.xml"))).isEmpty();
+    }
+
     private Node asNode(String path) throws Exception {
         try (InputStream stream = Thread.currentThread().getContextClassLoader().getResourceAsStream(path)) {
             return DocumentBuilderFactory.newInstance()
                 .newDocumentBuilder()
                 .parse(stream);
         }
     }
+
+    // TODO: write test with
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)